README.md

```diff
@@ -102,6 +81,37 @@ A build toolchain for Snyk Docker images.
 | snyk/snyk:sbt                | hseeberger/scala-sbt:8u212_1.2.8_2.13.0 |
 | snyk/snyk:scala              | hseeberger/scala-sbt:8u212_1.2.8_2.13.0 |
 
+#### Following images not supported by their vendors
+The following images will not be maintained in the future, followed by a deprecation process.
```

Comment:
I would use stronger language. "These images are not supported by their vendors and should no longer be used. Users of these images should move to a vendor-supported upstream image base immediately"

Reply:
I would also stop building them in this PR. Remove them from the actions. It wouldn't break anyone's workflow and takes a stronger security posture.